main.js

```diff
@@ -0,0 +1,78 @@
+// constant variable declare
```

Comment:
- [ ] I think it's better to create a collection that keeps a list of books (hint: you can use an array of objects for that). It's a project requirement. You are using the local storage to do this, but the local storage must be used to save your data for the next time you use the app.